ui/src/institutions/components/InstitutionsHistoricalDataList.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { List } from 'immutable';
+import InlineList, {
+  SEPARATOR_MIDDLEDOT,
+} from '../../common/components/InlineList';
+
+function InstitutionsHistoricalDataList({ historicalData }) {
+  return (
+    <InlineList
+      label="Note"
```

Comment:
so both `public_notes` and `historical_data` have same `label`?

Reply:
yes, I asked @StellaCh about this and she said to put ```Note``` for both